components/datetime/src/combo.rs

```diff
@@ -203,3 +203,60 @@ where
     type Z = Z::Z;
     type GluePatternV1 = datetime_marker_helper!(@glue, yes);
 }
+
+#[cfg(test)]
+mod tests {
+    use icu_calendar::Iso;
+    use icu_locale_core::locale;
+    use icu_time::{
+        zone::{IanaParser, UtcOffset, UtcOffsetCalculator},
+        DateTime, ZonedDateTime,
+    };
+    use writeable::assert_writeable_eq;
+
+    use crate::{
+        fieldsets::{zone::SpecificLong, YMDT},
+        DateTimeFormatter,
+    };
+
+    #[test]
+    fn please_dont_do_this() {
```

Comment:
nit: useless test module

```suggestion
#[test]
fn please_dont_do_this() {
    use icu_calendar::Iso;
    use icu_locale_core::locale;
    use icu_time::{
        zone::{IanaParser, UtcOffset, UtcOffsetCalculator},
        DateTime, ZonedDateTime,
    };
    use writeable::assert_writeable_eq;

    use crate::{
        fieldsets::{zone::SpecificLong, YMDT},
        DateTimeFormatter,
    };
```